### PR TITLE
README

### LINKED ISSUE #19: Add project information to the README
Update the project's README to include our names and description of project.

### PR DESCRIPTION
**Pull request by jpb7**
Adds some basic markdown to the Google Drive document like links and section headings.

### CODE REVIEW

File: stock-tracker/src/components/chart/CandleStick.js
Comment (srimel) [inline stock-tracker/src/components/chart/CandleStick.js:1]: Not sure why this file is included

File: stock-tracker/src/components/details/Details.js
Comment (srimel) [inline stock-tracker/src/components/details/Details.js:1]: This file also shouldn't be included

**Review by srimel — CHANGES_REQUESTED** [review]
Thanks for adding to our README!

I noticed old changes that are already included in main branch from CandleStick.js and Details.js are in this PR.

Can you push up without these files?

File: stock-tracker/src/components/chart/CandleStick.js
Comment (jpb7) [reply stock-tracker/src/components/chart/CandleStick.js:1]: Me neither...

**Comment by srimel** [comment]
dead branch

**Pull request closed without merging**